smarts/core/mission_planner.py

```diff
@@ -202,44 +202,49 @@ def _edge_ids(self, pose: Pose, lane_id: str = None):
         return edge_ids
 
     def cut_in_waypoints(self, sim, pose: Pose, vehicle):
+
+        aggressivenes = 5
         radius = self._mission.task.trigger_radius
         neighborhood_vehicles = sim.neighborhood_vehicles_around_vehicle(
-            vehicle=vehicle, radius=radius
+            vehicle=vehicle, radius=80
```

Comment:
This should either be reverted or the use of `trigger_radius` reworked or removed.

Reply:
Thanks, sorry, I removed the `trigger_radius`